experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -446,7 +446,7 @@ export class NodeSDK {
   }
 
   private configureLoggerProviderFromEnv(): void {
-    const logExportersList = process.env.OTEL_LOGS_EXPORTER ?? '';
+    const logExportersList = getStringFromEnv('OTEL_LOGS_EXPORTER') ?? '';
     const enabledExporters = filterBlanksAndNulls(logExportersList.split(','));
```

Comment:
I think this could use `getStringListFromEnv`, and then don't need to use `filterBlanksAndNulls`. See how a similar thing was done here: https://github.com/open-telemetry/opentelemetry-js/pull/5578/files

Reply:
Good idea, didn't see that one. I will have a look.

Reply:
@trentm I have updated it :)